tools/src/main/scala/scala/scalanative/codegen/Generate.scala

```diff
@@ -504,7 +592,7 @@ object Generate {
     }
   }
 
-  private object Impl {
+  object Impl {
```

Comment:
I need to move the constats that I need in to use in `Lower` outside of the `Impl` object and make them public.